packages/common/src/store/pages/token-dashboard/selectors.ts

```diff
@@ -36,3 +36,16 @@ export const getHasAssociatedWallets = (state: CommonState) => {
 }
 export const getRemoveWallet = (state: CommonState) =>
   state.pages.tokenDashboard.associatedWallets.removeWallet
+export const getCanRecipientReceiveWAudio = (state: CommonState) => {
+  if (
+    state.pages.tokenDashboard.modalState?.stage === 'SEND' &&
+    state.pages.tokenDashboard.modalState.flowState.stage ===
+      'AWAITING_CONFIRMATION'
+  ) {
+    return state.pages.tokenDashboard.modalState.flowState
+      .canRecipientReceiveWAudio
+  }
+  throw new Error(
```

Comment:
I don't think the selector should throw. We're also not handling this exception in the UI it doesn't look like.

Let's just pick a good default here?

Reply:
Good point!